std/experimental/allocator/package.d

```diff
@@ -573,11 +573,22 @@ unittest
 
 private T[] uninitializedFillDefault(T)(T[] array) nothrow
 {
-    static immutable __gshared T t;
+    T t = T.init;
     fillWithMemcpy(array, t);
     return array;
 }
 
+pure nothrow @nogc
```

Comment:
And by the way there should be a rule about this in the [Phobos styleguide](https://dlang.org/dstyle.html).